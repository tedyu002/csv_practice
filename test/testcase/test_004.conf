CSV_INPUT=test_004.conf.input
CSV_OUTPUT=test.out
CSV_ERROR=test.err
CSV_RESULT=test.res
HEADERS=H1 CHAR(4),H2 VARCHAR(32),H3 VARCHAR(12),H4 INTEGER,H5 BOOL,H6 DATETIME,H7 DOUBLE
CSV_FORMULA=Print H4[10]
CSV_FORMULA=Print H1[5]
SORT_HEADERS=H1,H3
SORT_ORDER=DESC
