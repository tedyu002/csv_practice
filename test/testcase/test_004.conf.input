"1997",Ford,E350,800000,TRUE,2017/06/03 12:30:01,12.35
1991,"Mercedes-Benz",S320,2800000,TRUE,2013/06/04 23:30:01,12.66
"2003","Mercedes-Benz","C220","1800000","TRUE","1998/10/03 08:33:23","528.69"
2016,Volvo,S60,1750000,FALSE,2017/06/09 03:49:00,52.63
"1991",Audi,A8,1850000,TRUE,2017/05/15 02:18:37,12.5
2003,Mazda,CX-5,1150000,FALSE,2017/05/15 02:18:52,45.96
2017,Skoda,Yeti,950000,FALSE,2017/04/13 02:18:52,12.52
2017,Skoda,Rapid,958000,TRUE,2017/05/15 07:47:34,15.63
2016,Skoda,Citigo,858000,FALSE,2017/05/15 07:48:37,172.8
1991,Audi,Q7,2250000,TRUE,2017/03/15 07:48:37,15.69
2016,Volvo,XC90,2550000,TRUE,2017/05/12 08:52:28,28.36
2016,Volvo,V40,1550000,FALSE,2017/05/15 09:08:12,69.63
2015,Volvo,S80,2350000,FALSE,2017/06/10 10:53:44,35.36
2016,Volvo,XC60,2350000,FALSE,2017/01/10 10:53:57,12.63
2016,Volvo,XC70,2550000,TRUE,2017/06/10 12:41:15,29.52
2017,BMW,X1,1550000,TRUE,2017/06/19 13:49:12,27.85
2015,BMW,X2,1650000,TRUE,2017/06/10 19:41:30,26.36
