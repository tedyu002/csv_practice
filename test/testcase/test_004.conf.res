2350000
"2016"
