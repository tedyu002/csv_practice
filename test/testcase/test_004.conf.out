"2017","Yeti","Skoda",950000,FALSE,2017/04/13 02:18:52,12.52
"2017","X1","BMW",1550000,TRUE,2017/06/19 13:49:12,27.85
"2017","Rapid","Skoda",958000,TRUE,2017/05/15 07:47:34,15.63
"2016","XC90","Volvo",2550000,TRUE,2017/05/12 08:52:28,28.36
"2016","XC70","Volvo",2550000,TRUE,2017/06/10 12:41:15,29.52
"2016","XC60","Volvo",2350000,FALSE,2017/01/10 10:53:57,12.63
"2016","V40","Volvo",1550000,FALSE,2017/05/15 09:08:12,69.63
"2016","S60","Volvo",1750000,FALSE,2017/06/09 03:49:00,52.63
"2016","Citigo","Skoda",858000,FALSE,2017/05/15 07:48:37,172.80
"2015","X2","BMW",1650000,TRUE,2017/06/10 19:41:30,26.36
"2015","S80","Volvo",2350000,FALSE,2017/06/10 10:53:44,35.36
"2003","CX-5","Mazda",1150000,FALSE,2017/05/15 02:18:52,45.96
"2003","C220","Mercedes-Benz",1800000,TRUE,1998/10/03 08:33:23,528.69
"1997","E350","Ford",800000,TRUE,2017/06/03 12:30:01,12.35
"1991","S320","Mercedes-Benz",2800000,TRUE,2013/06/04 23:30:01,12.66
"1991","Q7","Audi",2250000,TRUE,2017/03/15 07:48:37,15.69
"1991","A8","Audi",1850000,TRUE,2017/05/15 02:18:37,12.50
